cockatrice/src/client/ui/picture_loader.cpp

```diff
@@ -145,18 +145,20 @@ PictureLoaderWorker::PictureLoaderWorker()
     networkManager->setRedirectPolicy(QNetworkRequest::ManualRedirectPolicy);
     connect(networkManager, SIGNAL(finished(QNetworkReply *)), this, SLOT(picDownloadFinished(QNetworkReply *)));
 
-    cacheFilePath = "redirectCache.ini";
+    cacheFilePath = SettingsCache::instance().getRedirectCachePath() + "cacheSettings.ini";
```

Comment:
I'd call it just "cache.ini" personally, since the directory is called redirect. Frankly, do we need to put this into a folder, can we call it "redirectCache" again like you had and just drop it in the /pics folder, next to CUSTOM?